nuxeo-compound-documents/src/test/java/org/nuxeo/compound/documents/TestCompoundDocumentImporter.java

```diff
@@ -63,4 +70,61 @@ public void testCreateCompoundDocument() throws IOException {
         assertCompoundDocument(doc);
     }
 
+    @Test
+    public void testBadArchive() throws IOException {
+        Blob blob = getBadArchive();
+        FileImporterContext context = FileImporterContext.builder(session, blob, "/").build();
+
+        NuxeoException e = assertThrows(NuxeoException.class, () -> fileManager.createOrUpdateDocument(context));
+
+        String compoundDocName = FilenameUtils.removeExtension(blob.getFilename());
+        assertEquals(String.format("Failed to create CompoundDocument for archive: %s.zip in parent: %s",
+                compoundDocName, context.getParentPath()), e.getMessage());
```

Comment:
```suggestion
        assertEquals(String.format("Failed to create CompoundDocument for archive: %s in parent: %s",
                blob.getFilename(), context.getParentPath()), e.getMessage());
```
?

Didn't see but the Blobs returned by `getTestArchive` and `getBadArchive` should have a filename ending with `.zip` - like in the real world.

Reply:
Agree with @troger, the blob filename should be `test.zip` and not the file name.